projects/plugins/boost/app/features/optimizations/minify/cssmin/cssmin.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+// Simple loader file
+// Library downloaded directly from https://github.com/tubalmartin/YUI-CSS-compressor-PHP-port
```

Comment:
cssmin appears to use the [BSD license](https://github.com/rgrove/cssmin/blob/master/LICENSE), which requires the license be attached. I don't see the BSD license text for it in here?

Can we install cssmin via composer, to save adding it to our repository?

Reply:
Yep yep! Done - 24b679cde2a4af547ccadd7aa1166bd0224aa34d.